packer/builder/azure/driver_restapi/installer/main_test.go

```diff
@@ -3,6 +3,8 @@
 // Licensed under the Apache License, Version 2.0.
 // See License.txt in the project root for license information.
 
+// +build !ci
+
```

Comment:
I think you should use Test.Skip instead. otherwise you won't notice compilation errors in these files in CI.